cli/azd/cmd/pipeline.go

```diff
@@ -247,7 +252,12 @@ You can view the GitHub Actions here: https://github.com/%s/actions
 		return fmt.Errorf("prompting to push: %w", err)
 	}
 
-	if doPush {
+	// Check if GitHub actions are disabled *Only* when user requested to push changes AND this is NOT a just-created repo
+	//
+	// A repo that is just created would return zero GitHub Actions and might be confused by azd
+	// as a repo where Actions are disabled. Sadly, there's not GitHub API to fetch exact information
+	// to distinguish between disabled-after-fork v/s repo-disabled-actions v/s similar scenarios).
```

Comment:
Found this that seems to work:
```
PS C:\Users\weilim> gh api -H "Accept: application/vnd.github+json" /repos/weikanglim/weilim-todo-python/actions/permissions
{
  "enabled": true,
  "allowed_actions": "all"
}
PS C:\Users\weilim> gh api -H "Accept: application/vnd.github+json" /repos/weikanglim/weilim-todo-python/actions/permissions
{
  "enabled": false
}
```

From the [docs](https://docs.github.com/en/rest/actions/permissions#get-github-actions-permissions-for-a-repository):

Gets the GitHub Actions permissions policy for a repository, including whether GitHub Actions is enabled and the actions and reusable workflows allowed to run in the repository.

Reply:
Thank you Sir.
This API works for manually disabling or enabling actions from `/settings/actions` in a Repo.  But it can't detect when actions are disabled after forking :(